src/components/binding/binding-renderer.js

```diff
@@ -236,6 +234,9 @@ genotet.BindingRenderer.prototype.dataLoaded = function() {
     start: this.data.detailXMin,
     end: this.data.detailXMax
   });
+
+  // Initialize binding tracks in the panel.
+  this.signal('track');
```

Comment:
Are you firing this only at the startup of the system? Or are you assuming dataLoaded is called anytime loadCounter is decreased to zero?